include/seqan3/search/all.hpp

```diff
@@ -70,33 +112,17 @@
  *
  * \endif
  *
- * **Reference:**
- *
- * Kianfar, K., Pockrandt, C., Torkamandi, B., Luo, H., & Reinert, K. (2018).
+ * # Membership queries (lookups)
  *
- * Optimum Search Schemes for Approximate String Matching Using Bidirectional FM-Index. bioRxiv, 301085.
- * https://doi.org/10.1101/301085
- *
- *
- * \if KMER
- * ## K-mer index
- *
- * A k-mer index can be used to efficiently retrieve all occurrences of a certain k-mer in the text.
- * The k-mer can be either an exact string of length k or it can contain one or more wildcards,
- * which denote positions of arbitrary characters.
- *
- * An exact k-mer is represented as seqan3::ungapped, and wildcards can be defined with seqan3::shape.
- * Please check the respective documentation for details and examples.
- *
- * \note The k-mer index is not yet implemented.
- * \sa seqan3::views::kmer_hash
- * \sa seqan3::views::minimiser
- * \endif
+ * If you are not interested in the exact match of your query, but simply whether your query can be found or not
+ * (also called membership query or lookup), SeqAn offers efficient data structures for this purpose.
+ * Of course you could also use a simple hash map (e.g. `std::unordered_map`) but depending on your data
+ * this can quickly become infeasible in terms of memory consumption and runtime.
```

Comment:
```suggestion
 * If you are not interested in the exact match of your query, but simply whether your query can be found or not
 * (also called membership query or lookup), SeqAn offers efficient data structures for this purpose.
 * Of course you could also use a simple hash map (e.g., `std::unordered_map`) but depending on your data
 * this can quickly become infeasible in terms of memory consumption and runtime.
```